docs/contributor-docs/ansible-guide.md

```diff
@@ -25,7 +25,7 @@ If you are not familiar with the Ansible, it is recommended to start with their
 
 * Familiarity or knowledge of Ansible
 
-* Ansible control workstation, laptop or server that runs only MacOs, BSD or Linux with the following:
+* Ansible control personal computer or server that runs only MacOs, BSD or Linux with the following:
```

Comment:
This grammar doesn't flow -- Should it be `Ansible controlled`?

Reply:
I see - this is the `Ansible Control Node` - the machine where Ansible is installed.   Maybe this should say `Ansible Control Node - personal computer...`

Reply:
This was foreign to me. I looked it up, thanks to your comments and the link in the docs, and I have now created a clearer explanation of the Ansible Control Node and its requirements. done.